performance_regression_gradual_grow_throughput.py

```diff
@@ -202,67 +184,45 @@ def run_gradual_increase_load(self, stress_cmd_templ, stress_num, num_loaders, c
-                exception=exc
-            ).publish_or_dump()
+        perf_analyzer.check_regression(test_name=test_name, test_results=total_summary, test_details=setup_details)
+
+    def get_cs_distribution(self):
+        popuplation_distribution = CSPopulateDistribution(self.params.get("cs_populating_distribution"))
+        if popuplation_distribution == CSPopulateDistribution.GAUSS:
+            return f"GAUSSIAN(1..{self.ROW_COUNT},{self.GAUSS_CENTER},{self.GAUSS_SIGMA})"
+        elif popuplation_distribution == CSPopulateDistribution.UNIFORM:
+            return f"UNIFORM(1..{self.ROW_COUNT})"
+        else:
+            self.log.error("Unsupported cs population distribution")
+            return ""
 
     @staticmethod
-    def _calculate_average_max_latency(results):
+    def _calculate_average_latency(results):
         status = defaultdict(float).fromkeys(results[0].keys(), 0.0)
-        max_latency = defaultdict(list)
-
         for result in results:
             for key in status:
                 try:
                     status[key] += float(result.get(key, 0.0))
-                    if key in ["latency 95th percentile", "latency 99th percentile"]:
-                        max_latency[f"{key} max"].append(float(result.get(key, 0.0)))
                 except ValueError:
                     continue
 
         for key in status:
             status[key] = round(status[key] / len(results), 2)
 
-        for key, latency in max_latency.items():
-            status[key] = max(latency)
-
         return status
 
-    def warmup_cache(self, stress_cmd_templ):
-        stress_queue = []
-        for stress_cmd in stress_cmd_templ:
-            params = {"round_robin": True, "stats_aggregate_cmds": False}
-            stress_cmd_to_run = stress_cmd.replace("$threads", "1000")
-            # Run c-s command without throttle
-            stress_cmd_to_run = stress_cmd_to_run.replace("$throttle", "")
-            # TODO: get "n" from c-s command or do it configurable
-            stress_cmd_to_run = stress_cmd_to_run.replace(
-                "duration=20m", "n=216666669" if "gauss" in stress_cmd_to_run else "n=20000000")
-            # For mixed workflow run read warm up
-            stress_cmd_to_run = stress_cmd_to_run.replace("mixed", "read")
-            stress_cmd_to_run = stress_cmd_to_run.replace("cl=QUORUM", "cl=ALL")
-            params.update({'stress_cmd': stress_cmd_to_run})
-            # Run all stress commands
-            self.log.debug('RUNNING warm up stress cmd: %s', stress_cmd_to_run)
-            stress_queue.append(self.run_stress_thread(**params))
-
-        for stress in stress_queue:
-            self.get_stress_results(queue=stress, store_results=False)
+    def warmup_cache(self, compaction_strategy):
+        cmd = f"cassandra-stress read no-warmup cl=QUORUM duration=180m -pop 'dist={self.get_cs_distribution()}' -mode native cql3 -rate 'threads=500 throttle=35000/s'"  # pylint: disable=line-too-long
```

Comment:
I think we should put this into SCT configuration,

but those number of threads and throttling, is that from the original code ? or something you or @roydahan tried and verified ?